src/Microsoft.PowerShell.Commands.Management/commands/management/SetClipboardCommand.cs

```diff
@@ -44,6 +44,13 @@ public class SetClipboardCommand : PSCmdlet
         [Parameter]
         public SwitchParameter PassThru { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether to use OSC52 escape sequence to set the clipboard of host instead of target.
+        /// </summary>
+        [Parameter]
+        [Alias("ToLocalhost")]
+        public SwitchParameter AsOSC52 { get; set; }
```

Comment:
We should use best name for parameter and avoid alias at all.

Reply:
In this case, the WG decided that the parameter should be explicitly technical so that a user would easily search for it while the alias is for someone already familiar with this switch and its purpose